deploy/quickstart/quick_start.py

```diff
@@ -0,0 +1,245 @@
+import os
+from os import path
+import urllib.request
+from subprocess import call
+import subprocess
+
+
+def quick_start():
+    version = "v0.2.0"
```

Comment:
nit: We use single quotes in python everywhere else in this codebase, could you update throughout for consistency?